design/resources.go

```diff
@@ -324,6 +330,7 @@ var _ = Resource("trackerquery", func() {
 	})
 
 	Action("create", func() {
+		Security("jwt")
```

Comment:
Good catch. Yes, I will write a similar test for tracker query. Waiting for #366 to be merged.